oidc-ui/configure_start.sh

```
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+#installs the pre-requisites.
+set -e
+
+echo "Downloading pre-requisites started."
+
+# Check if $i18n_url_env is not empty
+if [[ -n "$i18n_url_env" ]]; then
+    echo "i18n_url_env is set: $i18n_url_env"
+    wget --no-check-certificate --no-cache --no-cookies $i18n_url_env -O $i18n_path/esignet-i18n-bundle.zip
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)